QualificationChecker/Controllers/QualificationClass.cs

```diff
@@ -0,0 +1,131 @@
+﻿using QualificationChecker.Models;
+
+namespace QualificationChecker.Controllers
+{
+    public abstract class QualificationManager
+    {
+        public Candidate Candidate { get; private set; }
+        public Organization Organization { get; private set; }
+        protected List<Candidate> QualifiedCandidates = new List<Candidate>();
+
+        protected QualificationManager(Candidate candidate, Organization organization)
+        {
+            Candidate = candidate;
+            Organization = organization;
+        }
+
+        // Method to tell if a Candidate is interested in the position
+        public bool IsCandidateInterestedInPosition()
+        {
+            if (Candidate.InterestedPositionIds == null)
+            {
+                return false;
+            }
+
+            return Candidate.InterestedPositionIds.Any(candidatePositionId =>
+                Organization.OrgQuestions.Any(OrgQuestion => OrgQuestion.PositionId == candidatePositionId));
```

Comment:
I think there is still a way to do this with dictionary lookups.

Reply:
I forgot I can refactor that